pangeo_forge_recipes/storage.py

```diff
@@ -25,11 +25,15 @@ def _get_url_size(fname, **open_kwargs):
     return size
 
 
-def _copy_btw_filesystems(input_opener, output_opener, BLOCK_SIZE=10_000_000):
+def _copy_btw_filesystems(input_opener, output_opener, BLOCK_SIZE=10_000_000, **kwargs):
+    streaming = True if kwargs.pop("block_size", True) == 0 else False
+    if os.getenv("PANGEO_FORGE_BLOCK_SIZE"):
+        BLOCK_SIZE = int(os.getenv("PANGEO_FORGE_BLOCK_SIZE"))
     with input_opener as source:
         with output_opener as target:
+            count = summed_bytes = 0
             while True:
-                logger.debug("_copy_btw_filesystems reading data")
+                logger.debug("_copy_btw_filesystems reading data") if not streaming else None
```

Comment:
I would rather have one single if / else block for the `streaming` parameter, so maybe we can just remove this line completely?